src/Gzero/Api/Controller/Admin/ContentController.php

```diff
@@ -202,15 +202,13 @@ public function update($id)
     /**
      * Removes the specified resource from database.
      *
-     * @param int  $id          Content id
-     *
-     * @param bool $forceDelete if true use forceDelete
+     * @param int $id Content id
      *
      * @return \Illuminate\Http\JsonResponse
      */
-    public function destroy($id, $forceDelete = false)
+    public function destroy($id)
     {
-        $forceDelete = filter_var($forceDelete, FILTER_VALIDATE_BOOLEAN);
+        $forceDelete = \Input::has('force');
```

Comment:
I think that we should check if the force is to to true, not only if it exists.
The has method returns true if the value is present and is not an empty string, so if you type for eg. force=abxdsada it will be valid in this case.

I'd rather be sure that user type force=true